docs/mkdocs.yml

```diff
@@ -21,26 +21,11 @@ plugins:
   - search
 
 nav:
-  - index.md
-  - Concepts:
```

Comment:
I don't think these can be moved out of iceberg-java versioned docs since they contain descriptions and configs specific to iceberg-java.